client/components/review.js

```diff
@@ -10,6 +10,7 @@ import flatten from 'lodash/flatten';
 import values from 'lodash/values';
 import isUndefined from 'lodash/isUndefined';
 import isNull from 'lodash/isNull';
+import { LATEST, GRANTED } from '../constants/change';
```

Comment:
Are these constants actually used anywhere else? It seems like they're just references within this one file, so I probably wouldn't abstract them away like this.

Reply:
yes, in application-summary